pkg/nvidia/scripts/process-cdi.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
```

Comment:
So is the purpose of `process-cdi.sh` to be run during `pkg/nvidia` build (i.e. `lfedge/eve-nvidia` image), when it reads the CDI yaml files, and uses that to find any binaries it needs in the `.deb` files and place them in the final image, so they are available to pillar when needed?

Reply:
Exactly this.

Reply:
Can we mark that, both here and in the README? So people can glance at it and know how things work?